### PR TITLE
Jetpack: update to-test.md for 10.8

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

Update testing instructions for Jetpack 10.8-beta

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Proofread.

### CODE REVIEW

**Comment by samiff** [comment]
I left two PRs out that had the Needs Testing label:

* **PHPCS: Protect** #23509 - Not sure what specific testing instructions would entail.
* **Unified payments block selector** #22491 - Requires a paid plan to actually see more than one payment option, and don't want to crowd the testing instructions.

Also of note is the omission of **Add Resumable Upload Support to the VideoPress Block** #22584 - This feature is live for wpcom simple sites only at the moment. Hopefully the feature is available to all Jetpack site in the future at which pointed we could could give it another test.

File: projects/plugins/jetpack/to-test.md [hunk 1]
```diff
@@ -1,88 +1,64 @@
...
-- Business Hours
+### Jetpack Block Settings Discoverability
 
-Alignment options were also added to:
+We've improved the discoverability of Jetpack Blocks in settings and added the ability to toggle them if desired:
 
-- Repeat Visitor blocks
+- Navigate to `Jetpack > Settings`.
+- Click on the Search icon and search for "blocks".
+- You should see a toggle setting with "Jetpack Blocks give you the power to deliver quality content..."
```
Comment (jeherve) [inline projects/plugins/jetpack/to-test.md:46]:
```suggestion
- You should see a toggle setting with "Jetpack Blocks give you the power to deliver quality content..."
- That toggle **must** be active by default.
```

**Review by jeherve — CHANGES_REQUESTED** [review]
This looks good, I only have a tiny suggestion.

**Review by jeherve — APPROVED** [review]

**Comment by jeherve** [comment]
If we end up merging #23362, I think it would make sense to add it to the testing list for this release.

**Pull request merged**